packages/webapp/src/pages/Activity.tsx

```diff
@@ -458,21 +335,32 @@ export default function Activity() {
         }
     }
 
+    if (error || logActivitiesError) {
+        requestErrorToast();
```

Comment:
it shouldn't not be triggered by the rendering, otherwise it will pop each time you render the view.
<img width="1304" alt="Screenshot 2024-02-23 at 15 12 38" src="https://github.com/NangoHQ/nango/assets/1637651/53dcd7e3-a47b-400a-8515-af6753392913">